seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/master/JobHistoryService.java

```diff
@@ -73,6 +82,8 @@ public class JobHistoryService {
      */
     private final Map<Long, JobMaster> runningJobMasterMap;
 
+    private final Map<Long, JobMaster> finishedJobMasterMap;
```

Comment:
I prefer store `historyExecutionPlan` into  `JobDAGInfo`. Not store all JobMaster.